source/_integrations/ness_alarm.markdown

```diff
@@ -23,18 +25,55 @@ The `ness_alarm` {% term integration %} will allow Home Assistant users who own
 
 There is currently support for the following device types within Home Assistant:
 
-- Binary sensor: Reports on zone statuses
+- Binary sensor: Reports on zone statuses with configurable device classes
 - Alarm control panel: Reports on alarm status, and can be used to arm/disarm the system
 
-The module communicates via the [Ness D8x/D16x ASCII protocol](https://ia802202.us.archive.org/16/items/ness-d-8x-d-16x-serial-interface.-ascii-protocol/Ness%20D8x%20D16x%20Serial%20Interface.%20ASCII%20Protocol.pdf).
+The module communicates via the [Ness D8x/D16x ASCII protocol](https://github.com/nickw444/nessclient/blob/master/D8-32X%20Serial%20Protocol%20Public.pdf).
+
+{% include integrations/config_flow.md %}
 
 ## Configuration
 
-A `ness_alarm` section must be present in the {% term "`configuration.yaml`" %} file and contain the following options as required:
-{% include integrations/restart_ha_after_config_inclusion.md %}
+The Ness Alarm integration is configured through the UI. To add the integration:
+
+1. Go to **Settings** → **Devices & Services**
+2. Click **+ Add Integration**
+3. Search for **Ness Alarm**
+4. Follow the setup steps
+
+During setup, you'll need to provide:
+
+- **Host**: The hostname or IP address of your IP232 module
+- **Port**: The port on which the IP232 module listens (default: 2401)
+- **Max Zone Count**: Maximum number of zones your panel supports (default: 16)
+- **Scan Interval**: Time between updates in seconds (default: 60)
+- **Infer Arming State**: Enable workaround for panels with firmware < v5.8
+- **Support Home Arm**: Enable ARM_HOME functionality
```

Comment:
This section could be covered in a [configuration basic block](https://developers.home-assistant.io/docs/documenting/integration-docs-examples#configuration_basic-block).